wandb/cli/cli.py

```diff
@@ -1417,10 +1417,37 @@ def docker(
     "--upgrade", is_flag=True, default=False, help="Upgrade to the most recent version"
 )
 @click.option(
-    "--edge", is_flag=True, default=False, help="Run the bleading edge", hidden=True
+    "--edge", is_flag=True, default=False, help="Run the bleeding edge", hidden=True
 )
 @display_error
-def local(ctx, port, env, daemon, upgrade, edge):
+def local(*args, **kwargs):
+    wandb.termwarn(
+        "`wandb local` has been replaced with `wandb server`."
+    )
+    wandb.termlog(
+        "Please use `wandb server` instead."
+    )
```

Comment:
wait do we want to keep both? that's how I read @raubitsj comment and only in the future move to `server` i'm fine doing it now... the only issue is if someone have some scripts that use `wandb local` they might break

Reply:
Agreed with @kptkin here.  There's no reason to break the old command.  Let's print the warning but just call `wandb server`:

```
def local(ctx, *args, **kwargs):
    wandb.termwarn(
        "`wandb local` has been replaced with `wandb server`."
    )
    ctx.invoke(server, *args, **kwargs) 
```